src/PavValHackathon.Web.API/v1/Contracts/WeatherForecastContract.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace PavValHackathon.Web.API.v1.Contracts
+{
+    [SwaggerDiscriminator("WeatherForecast")]
+    public record WeatherForecastContract
```

Comment:
nigga what

Reply:
weather? wtf

Reply:
ahh, ignore the above. It was generated by a default template.